README.md

```diff
@@ -44,18 +44,65 @@ As an alternative to the above online version, the standalone server of BRouter
 
 2.  download one or more [data file(s)](https://brouter.de/brouter/segments4/) (rd5) into `segments4` directory
 
-### Configure BRouter-Web
+#### Configure BRouter-Web
 
 In the `brouter-web` subdirectory:
 
 1.  copy `config.template.js` to `config.js`
 2.  add your API keys (optional)  
     copy `keys.template.js` to `keys.js` and edit to add your keys
 
-### Run
+#### Run
 
 1.  start `./run.sh`
 
+### Running as Docker container (client only)
+
+brouter-web can be run as a Docker container, making it easy for continous deployment or running locally
+without having to install any build tools.
+
+The `Dockerfile` builds the application inside a NodeJS container and copies the built application into a
+separate Nginx based image. The application runs from a webserver only container serving only static files.
+
+#### Prerequisites
+
+- Docker installed
+- working directory is this repository
+- `config.template.js` copied to `config.js` and modified with a Brouter server, see `BR.conf.host`
+- `keys.template.js` to `keys.js` and add your API keys
+- Optionally create `profiles` directory with `brf` profile files and add path to `config.js`: 
+    
+      BR.conf.profilesUrl = 'profiles/';
+
+
+#### Building Docker image
+
+To build the Docker container run:
+
+      docker build -t brouter-web .
+
+This creates a Docker image with the name `brouter-web`.
+
+#### Running Docker container
+
+To run the previously build Docker image run:
+
+      docker run --rm --name brouter-web \
+        -p 127.0.0.1:80:80 \
```

Comment:
Shouldn't this map to 8080 instead of 80, as mentioned later?
```suggestion
        -p 127.0.0.1:8080:80 \
```

Reply:
Yes, that is a typo.